frontend/src/components/SettingsSidebar/index.jsx

```diff
@@ -169,14 +179,16 @@ export default function SettingsSidebar() {
                     </>
                   }
                 />
-                <Option
-                  href={paths.settings.security()}
-                  btnText="Security"
-                  icon={<Lock className="h-5 w-5 flex-shrink-0" />}
-                  user={user}
-                  flex={true}
-                  allowedRole={["admin", "manager"]}
-                />
+                {!multiUserModeEnabled && (
+                  <Option
+                    href={paths.settings.security()}
+                    btnText="Security"
+                    icon={<Lock className="h-5 w-5 flex-shrink-0" />}
+                    user={user}
+                    flex={true}
+                    allowedRole={["admin", "manager"]}
+                  />
+                )}
```

Comment:
Lets have a new property on the Option that allows us to set it as hidden with  a boolean property